wiki/Tournaments/Official_support/en.md

```diff
@@ -0,0 +1,165 @@
+- All multiplayer games relevant to the tournament must be created with the ```!mp make``` command, so that they do not expire.
+- All matches undertaken for the tournament must be recorded and their history made publicly available on the original forum post for the tournament in a clearly marked link in normal size font or larger.
+- In the event that a player becomes restricted during the course of the tournament, you may need to replay certain matches where they were involved in order to preserve the competitive integrity of the tournament.
+    - Organizers should contact the Support Team to receive information and guidance on these (hopefully rare) cases.
+
+### After the tournament
+
+Once the tournament has concluded, the organizer will need to submit the following to the osu! Support Team:
+
+- A list of applicable badge recipients (aka: the winners of your tournament)
+- A link to the publicly viewable catalogue of your tournament's match history
+
+LAN tournaments are also required to add links to VODs and a copy of the bracket of the entire tournament in order to be considered for a badge prize.
+
+## Frequently Asked Questions (FAQ)
+
+### Why does a tournament have to run at least twice or more in order to qualify for profile badges without experienced staff present?
+
+The second iteration of a tournament is often free of a lot of the organizational issues of the first tournament, and has established a solid workflow and staff involved in its production. They are far less likely to peter out or be subject to questionable choices. The tournament is also far more identifiable if it runs more than just once.
+
+For all of these reasons (and more), we ask that profile badge prizes are only awarded to tournaments that have successfully run at least once before, unless a sizable majority of the team is experienced and accomplished at running tournaments in the past.
+
+### What is considered 'staff' as far as tournament management goes?
+
+We consider referees, mappool selectors, judges, team organizers, or any other form of managerial position directly involved in the maintenance and operation of a tournament to be a part of its staff.
+
+Commentators are not considered by us as staff, and may play or interact in tournaments freely so long as they are not directly involved any managerial aspect of running the tournament or violate anything mentioned above.
+
+We have historically had issues where unscrupulous organizers have geared tournaments specifically (ie: picked maps they're familiar with, adjusted seeds/brackets, etc) to farm profile badges for them and their friends. While we understand that most groups will not do this, we must enforce this restriction to keep things fair for everyone.
+
+### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems?
```

Comment:
shouldn't there be a clause here about mappool showcase replays/playtesting? this has been a question i've been asked several times by newer hosts where i always concluded with "OWC does it, so it should be fine". it's a bit confusing as mappooling is something usually only managed by the higher ups in staff but since refereeing after being eliminated is not allowed, having an influence on the mappool also shouldn't but as mentioned before OWC (and possibly other world cups which i haven't checked) get eliminated players to help

Reply:
oops just noticed i commented on more lines than needed, this was meant for only line 136 (im github noob sorry)

Reply:
I personally don't understand the rule that eliminated players cannot join staff afterwards. Sounds over-cautious.

Reply:
I would argue that joining the staff post-elimination looks bad in general but I guess we can have a debate about this. I personally have no issues in regards to having them being play-testers or replay makers.

To join as map selectors or refs I don't like the idea as much because, as former players, they could affect the outcome of the tournament (by picking stuff against teams they lost for example, its silly but possible. Schedule teams to bad times post-elimination is another example).

Reply:
Nothing stops them from not playing at all and entering staff in another tournament and rig it there as well; since hardly anyone ever plays only one tourney in their life.  
I do not think removing this limitation dramatically increases the chances of such bad behavior to occur.

I believe we can trust tournament hosts with this - even more since we now have a proper protocol for players involved to report such malicious behavior with the mentioned form.

Reply:
In my opinion, removing this limitation would change absolutely nothing.
I've seen it being broken in multiple tournaments before, and they've all worked just fine.
Feels just like a worse way of getting less staff members at this point. (Gamemodes other than standard, where tournament players are also active staff members can be a good example)
Especially with the reporting form at this point.
If there IS an actual issue with that, which I heavily doubt, then people can report it.
If there is an overwhelming amount of reports that shows this was a good idea in the first place, sure, let's put it back then.
But for now, it just feels like a limitation to staff members.

And also, you've brought up referees with this: "To join as map selectors or refs I don't like the idea as much because, as former players, they could affect the outcome of the tournament (by picking stuff against teams they lost for example, its silly but possible. Schedule teams to bad times post-elimination is another example)."

But have referees had this much power that they could rig an entire match by themselves?
I have geniunely not seen that in years.
And your first point applies to map selectors, which I don't think most map selectors nowadays do but okay, fair point.
Scheduling teams to bad times however?
In what kind of tournament are referees scheduling matches even?
I have geniunely never understood why players cannot be referees, since, if they are referees: - They don't ref their own match
- If they do blatant rigging, players are just gonna report it to the host, and the host will just overwrite their decision if it's obviously wrong.

Reply:
Updates to the article have been pushed, but this has been left completely unaddressed.

Is there any update? Especially as many tournaments, including osu! World Cups, go against that rule with playtesters?

Reply:
another bump cause im also interested in what the take is with players that sign up but arent guaranteed to be on a team (for example world cups)